internal/controllers/machineset/machineset_preflight.go

```diff
@@ -152,7 +151,7 @@ func (r *Reconciler) controlPlaneStablePreflightCheck(controlPlane *unstructured
 		return nil, errors.Wrapf(err, "failed to perform %q preflight check: failed to check if %s %s is provisioning", clusterv1.MachineSetPreflightCheckControlPlaneIsStable, controlPlane.GetKind(), cpKlogRef)
 	}
 	if isProvisioning {
-		return ptr.To(fmt.Sprintf("%s %s is provisioning (%q preflight failed)", controlPlane.GetKind(), cpKlogRef, clusterv1.MachineSetPreflightCheckControlPlaneIsStable)), nil
+		return ptr.To(fmt.Sprintf("%s %s is provisioning (%q preflight check failed)", controlPlane.GetKind(), cpKlogRef, clusterv1.MachineSetPreflightCheckControlPlaneIsStable)), nil
```

Comment:
I had the same thought. But I think the conditions won't have escape characters. We just have them here in go :)

Reply:
Example:
```bash
      - lastTransitionTime: "2024-11-08T14:35:25Z"
        message: Scaling up from 0 to 1 replicas is blocked because KubeadmControlPlane
          default/capi-quickstart-control-plane is provisioning ("ControlPlaneIsStable"
          preflight check failed); MachineSet version (1.30.4) and ControlPlane version
          (1.31.0) do not conform to kubeadm version skew policy as kubeadm only supports
          joining with the same major+minor version as the control plane ("KubeadmVersionSkew"
          preflight check failed)
        observedGeneration: 1
        reason: ScalingUp
        status: "True"
        type: ScalingUp
```